src/main/resources/hudson/plugins/plot/CSVSeries/config.jelly

```diff
@@ -3,9 +3,8 @@
          xmlns:t="/lib/hudson" xmlns:f="/lib/form">
     <f:entry>
         <div>
-            <f:entry title="${%Data series file}" field="file" help="/plugin/plot/help-series.html">
-                <f:textbox name="file" value="${series.file}"
-                           checkUrl="'descriptorByName/hudson.plugins.plot.PlotPublisher/checkSeriesFile?value='+toValue(this)"/>
+            <f:entry title="${%Data series file}" field="seriesFile" help="/plugin/plot/help-series.html">
```

Comment:
Have you tested all of these occurrences? These do not always get automagically wired to the descriptor methods as I've recently discovered.

Reply:
Thanks for pointing this out. I assumed those were working the same way, but it turns out this is needed to compatibility with Pipeline #32 

I reverted back t the checkDependsOn approach